src/internal/task/task_asyncify.go

```diff
@@ -45,6 +45,7 @@ type stackState struct {
 
 // start creates and starts a new goroutine with the given function and arguments.
 // The new goroutine is immediately started.
+//go:noinline
```

Comment:
Many places in interp assume no inlining has yet happened, so I think it's safe to leave out these `//go:noinline` pragmas.

Reply:
Alright yeah I just removed these.